src/test/java/StringTest.java

```diff
@@ -0,0 +1,54 @@
+import static org.assertj.core.api.Assertions.*;
+
+import java.util.Arrays;
+import java.util.List;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+public class StringTest {
+    @DisplayName("문자열 1,2 을 ,로 split 했을 때 1과 2로 분리된다")
+    @Test
+    void split_when_1_2() {
+        // given
+        String testStrings = "1,2";
+        // when
+        List<String> splitStrings = Arrays.asList(testStrings.split(","));
```

Comment:
```suggestion
        final String[] actual = "1,2".split(",");
```

로컬 변수 없이 구현해도 되지 않을까?